File: TrueVote.Services/Database/Glas.cs
using System;
using System.Collections.Generic;

namespace TrueVote.Services.Database;

public partial class Glas
{
    public int Id { get; set; }

    public int KorisnikId { get; set; }

    public int KandidatId { get; set; }

    public DateTime VrijemeGlasanja { get; set; }

    public bool Obrisan { get; set; }

    public virtual Kandidat Kandidat { get; set; } = null!;

    public virtual Korisnik Korisnik { get; set; } = null!;
}

File: UI/truevote_desktop/lib/models/glas.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:truevote_desktop/models/kandidat.dart';
import 'package:truevote_desktop/models/korisnik.dart';

part 'glas.g.dart';

@JsonSerializable()
class Glas {
  int id;
  int? korisnikId;
  int? kandidatId;
  DateTime? vrijemeGlasanja;
  Kandidat? kandidat;
  Korisnik? korisnik;

  Glas({
    required this.id,
    this.korisnikId,
    this.kandidatId,
    this.vrijemeGlasanja,
    this.kandidat,
    this.korisnik,
  });

  factory Glas.fromJson(Map<String, dynamic> json) => _$GlasFromJson(json);

  Map<String, dynamic> toJson() => _$GlasToJson(this);
}